docs/conf.py

```diff
@@ -322,5 +322,6 @@ def setup(app):
     'v1.17.0': ('https://www.envoyproxy.io/docs/envoy/v1.17.0', None),
     'v1.17.1': ('https://www.envoyproxy.io/docs/envoy/v1.17.1', None),
     'v1.17.2': ('https://www.envoyproxy.io/docs/envoy/v1.17.2', None),
-    'v1.18.0': ('https://www.envoyproxy.io/docs/envoy/v1.18.2', None)
+    'v1.18.0': ('https://www.envoyproxy.io/docs/envoy/v1.18.2', None),
+    'v1.18.0': ('https://www.envoyproxy.io/docs/envoy/v1.18.3', None)
```

Comment:
this needs to be `1.18.3` also if you want to refer to this version as such

Reply:
Ok.